frontend/catalyst/autograph/ag_primitives.py

```diff
@@ -271,7 +271,7 @@ def for_stmt(
     #   -> this will raise a warning to allow users to correct mistakes and allow the conversion
     #      to succeed, for example because they forgot to use a list instead of an array
 
-    # pylint: disable=multiple-statements,missing-class-docstring
+    # pylint: disable=missing-class-docstring
```

Comment:
Seems like this was silencing a warning.
```suggestion
    # pylint: disable=multiple-statements,missing-class-docstring
```

Reply:
I was getting a codefactor error that said that I was silencing a pylint warning that didn't need to be silenced, which is why I removed it.

Reply:
Very odd that codefactor is still complaining (especially because pylint is not complaning).

Reply:
Can you restore the silencing? :)